.gitignore

```diff
@@ -1,24 +1,45 @@
-*.pyc
-.idea/
+int main()
+
+*.pyc_file
+.idea/Trademarks_And
```

Comment:
@Queera - its unclear what the point of these changes is, they seem to be specific to your use case rather than normal build files we do not want to include for DeepSpeed.  Could you let us know/update the PR description with some clarification of what you're trying to accomplish here?